src/monitors.ts

```diff
@@ -73,8 +95,8 @@ export function buildMonitorParams(monitor: Monitor, cloudFormationStackId: stri
  * @param serverlessMonitorId - Unique ID string defined for each monitor
  * @returns true if a given monitor is a serverless recommended monitor
  */
-function checkIfRecommendedMonitor(serverlessMonitorId: string) {
-  return Object.keys(SERVERLESS_MONITORS).includes(serverlessMonitorId);
+function isRecommendedMonitor(serverlessMonitorId: string, recommendedMonitors: RecommendedMonitors) {
+  return Object.keys(recommendedMonitors).includes(serverlessMonitorId);
```

Comment:
You could simply do the below..
```suggestion
  return recommendedMonitors[serverlessMonitorId] !== undefined;
```
Executing `object.keys` and then running `.includes` is a roundabout way of doing what you're trying to do because it has to do two iteration operations..
1) `object.keys` should build and return an array of all the key names
2) `.includes` operation would have to iterate through the values of that array and string match each value against the ID

Whereas the suggested change will simply use the direct object key lookup that's the nature of a JSON object and is much more efficient